micro-rdk-ffi/micrordklib-idf-component/CMakeLists.txt

```diff
@@ -0,0 +1,39 @@
+idf_component_register(INCLUDE_DIRS "${CMAKE_CURRENT_SOURCE_DIR}")
+idf_build_set_property(INCLUDE_DIRECTORIES "${CMAKE_CURRENT_BINARY_DIR}/assets" APPEND)
+
+set(LIBMICRORDK_VERSION v0.2.1-rc1)
```

Comment:
I don't like this either. But I want the idf_component version to be strictly tied to a libmicrordk version otherwise it will be hard to understand which versions user are running. Also I don't know if cmake is smart enough to automatically pull the latest version without erasing the build directory & rebuilding.